### PR TITLE
GitHub-35: Fixing link reference and adding real link

### LINKED ISSUE #35: Update the 'Reference to Red Hat guides' example
This example doesn't follow the conventions for references that outlined in the IBM Style Guide (see Citations and references, General guidelines).

I suggest rewording the example 
from: See the JBoss EAP _Getting Started Guide_ for more information.
to: For more information, see the JBoss EAP _Getting Started Guide_.

### PR DESCRIPTION
**Pull request by bergerhoffer**
Fixes #35 

Also added a real link, so that the link goes somewhere instead of 404ing.

### CODE REVIEW

**Review by ncbaratta — APPROVED** [review]
lgtm

**Comment by pmkovar** [comment]
I think this can be safely merged, thanks.
